content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,175 @@
+- :guilabel:`Extra advantages...`: if any extra advantages are offered to the applicant, enter it in
+  the :guilabel:`Extra advantages...` field to the right of the :guilabel:`Proposed Salary` field.
+  This should be short and descriptive, such as `unlimited sick time` or `retirement plan`.
+
+Application Summary tab
+-----------------------
+
+Any additional details or notes that should be added to the applicant's card can be typed into this
+field.
+
+Skills tab
+----------
+
+Skills can be added to the applicant's card. For details on adding skills, refer to the
+:ref:`Create new employees <employees/skills>` document.
+
+.. image:: add-new-applicants/new-applicant.png
+   :align: center
+   :alt: All the fields for a new applicant form entered.
+
+.. _recruitment/quick-add-applicant:
+
+Quick add
+=========
+
+Navigate to the main recruitment dashboard by going to :menuselection:`Recruitment app -->
+Applications --> By Job Positions`. Click the :guilabel:`(#) New Applications` button on the job
+position card that the applicant should be added to. Then, click the on the :guilabel:`Quick Add`
+button, represented by a small :icon:`fa-plus` :guilabel:`(plus)` icon in the top-right corner of
+the :guilabel:`New` stage to quickly add a new applicant.
+
+Enter the following information on the card:
+
+- :guilabel:`Subject/Application`: enter the title for the card. Typically, this is the applicant's
+  name, and job position being applied to. For example: `Laura Smith - HR Manager`. This field is
+  **not** visible in the Kanban view of the :guilabel:`Applications` page for the job position,
+  unless the :guilabel:`Applicant's Name` is left blank. If there is no :guilabel:`Applicant's
+  Name`, then the :guilabel:`Subject/Application Name` is what is displayed on the applicant card.
+- :guilabel:`Applicant's Name`: enter the applicant's name.
+- :guilabel:`Email`: enter the applicant's email address.
+- :guilabel:`Applied Job`: the current job position populates this field. If needed, the job
+  position can be changed by selecting a different position from the drop-down menu. If a different
+  job position is selected, after the card is created, the card appears on the
+  :guilabel:`Applications` page for that newly-selected job position.
+
+After the information is entered, click :guilabel:`Add`. The applicant appears in the list, and a
+new blank applicant card appears.
+
+When doing a quick add, clicking away from the card, or clicking the :icon:`fa-trash-o`
+(:guilabel:`trash`) icon, discards the applicant.
+
+If preferred, after entering the :guilabel:`Applicant's Name` in the Kanban card that appears, click
+:guilabel:`Edit`, and a detailed applicant form loads. Refer to the :ref:`New applicant form
+<recruitment/applicant-details>` section for details about filling out the form.
+
+.. image:: add-new-applicants/quick-add.png
+   :align: center
+   :alt: All the fields for a new applicant form entered when using the Quick Add option.
```

Comment:
See my previous suggestion to move this section above the **New applicant form** section
```suggestion
```